### DISCUSSION TITLE
How to do sync function call in v 13?

### DISCUSSION THREAD

**Discussion by iamdineshkumar** [appwrite/appwrite#2912] [Q&A] [answered]
I am glad that functions can be work syncronously and asynchronously. But whether client sdk's released with this features? In document secion of create execution its mentioned as "Once this endpoint is called, your function execution process will start asynchronously." but how to get sync outcome of function without using get execution endpoint.

**Comment by brandonroberts** [comment, marked as answer]
The client SDK's have been updated to support 0.13. If you pass `async` as `false` it will run the function synchronously and wait until it's finished.
